pkg/controller/certificates/controller.go

```diff
@@ -181,6 +187,51 @@ func (cc *CertificateController) maybeSignCertificate(key string) error {
 	}
 	csr := obj.(*certificates.CertificateSigningRequest)
 
+	func() {
+		if !cc.approveAllKubeletCSRs ||
+			IsCertificateRequestApproved(csr) ||
+			!IsCertificateRequestDenied(csr) {
+			return
+		}
+
+		isKubeletBootstrapGroup := false
+		for _, g := range csr.Spec.Groups {
+			if g == "system:kubelet-bootstrap" {
+				isKubeletBootstrapGroup = true
+				break
+			}
+		}
+		if !isKubeletBootstrapGroup {
+			return
+		}
+
+		x509cr, err := utilcertificates.ParseCertificateRequestObject(csr)
+		if err != nil {
+			glog.Errorf("unable to parse csr %q: %v", csr.ObjectMeta.Name, err)
+			return
+		}
+		if !strings.HasPrefix(x509cr.Subject.CommonName, "system:node:") {
+			return
+		}
+		isNodeGroup := false
+		for _, g := range x509cr.Subject.Organization {
```

Comment:
Yes, [kublet sets organisation to `system:nodes`](https://github.com/kubernetes/kubernetes/blob/a41e6e3817666cb70fb1b8155959be6dd5069a99/cmd/kubelet/app/bootstrap.go#L194-L197).